content/projects/jam/index.adoc

```diff
@@ -10,49 +10,46 @@ links:
-* Promote - We give every JAM continuous social media love via Twitter, blog posts, Facebook, newsletter, and post on the jenkinsci.org events calendar.
-* Jenkins Expertise - We will help you to schedule a Jenkins expert as the key speaker at your JAM, or provide you with presentation materials.
+* Swag - We will send you cool swags (stickers, t-shirts, etc) to share with members
+* Promote - We give every CDF Meetup and JAM social media love via Twitter, blog posts, Facebook, newsletter, and post on the link:https://cd.foundation/events/list/[events calendar].
+* CI/CD Expertise - We will help you to schedule a project expert as the key speaker at your meetup, or provide you with presentation materials.
 * Outreach - We will assist in reaching out to your local community for food/venue sponsorship.
+* Support specific to JAMs - As CDF will manage the operational aspect of JAMs and CI/CD meetups, the Advocacy & OutReach SIG is a channel to provide guidance/advice to new JAM organizers if needed. Email jenkins-advocacy-and-outreach-sig@googlegroups.com 
 
 === Best Practices
 
-* Attend existing meetup groups to gauge Jenkins interest/market within your area.
-* Reach out to existing JAM organizer(s) to learn best practices and time commitment.
-* Set up a Twitter account (for example, #JenkinsNYC). Email your group’s Twitter ID to jenkinsci-jam@googlegroups.com so we can help to promote your JAM.
-* Promote, tweet and post your JAM in advance.
+* Attend existing meetup groups to gauge any of CDF's current projects interest/market within your area.
+* Reach out to existing CDF/JAM organizer(s) to learn best practices and time commitment.
+* Set up a Twitter account (for example, #JenkinsNYC). Email your group’s Twitter ID to meetups@cd.foundation so we can help to promote your meetup.
+* Promote, tweet and post your meetup in advance.
 * Avoid product pitches. Keep it real, it’s not about promotions. This is about - and for - learning opportunities for the community.
 * Shake up the format. Have presentations, panel discussions, roundtables, workshops, hackathons, etc.
-* Set a consistent date, time and location so members will plan for it. Avoid cancelling your meeting as people will lose faith and stop RSVPing for your JAM.
+* Set a consistent date, time and location so members will plan for it. Avoid cancelling your meeting as people will lose faith and stop RSVPing for your meetup.
+* Review the CDF meetups repository on Github https://github.com/cdfoundation/meetups
 
 === Stepping Down
 
-In the event you can no longer be a JAM organizer, we ask that you nominate a
+In the event you can no longer be an organizer, we ask that you nominate a
 replacement organizer in your place. If this is not possible please send email
-to jenkinsci-jam@googlegroups.com, we will work with you to find a replacement.
+to meetups@cd.foundation, we will work with you to find a replacement.
```

Comment:
Advocacy&Outreach for Jenkins meetups? No strong opinion, because we will be looped in anyway